pkg/operator/controllers/imageconfig/image_controller.go

```diff
@@ -26,42 +26,35 @@ import (
 )
 
 const (
-	ControllerName = "ImageConfig"
+	controllerName = "ImageConfig"
 	// Kubernetes object name
 	imageConfigResource = "cluster"
+	controllerEnabled   = operator.ImageConfigEnabled
```

Comment:
Question for me - when would we choose to have the indirection here (having the package-level constant) versus not? It looks like the value is only used once.